Poker.BE/Poker.BE.Domain/Game/GameConfig.cs

```diff
@@ -1,217 +0,0 @@
-﻿using Poker.BE.Domain.Game;
-using System;
-using System.Collections.Generic;
-
-namespace Poker.BE.Domain.Game
-{
-    public class GameConfig
```

Comment:
❓ i understand we removing `GameConfig` - is there any impact on other layers? (like Data Layer, Service Layer) from this removal?